tests/vagrant/boxes/tumbleweed.pm

```diff
@@ -50,7 +50,7 @@ sub run() {
     #
     # get Vagrantfile template and replace the distro name & insert box filenames
     #
-    assert_script_run("wget --quiet " . data_url("virtualization/Vagrantfile"));
+    assert_script_run("wget --quiet " . data_url("vagrant/Vagrantfile"));
```

Comment:
I would rather recommend to move your files to `virtualization/vagrant` so that the scope is clearer, the same applies for the .pm files. At this moment most of the directories under `/tests/` help to organize the test modules by domain (although this seems to be an unwritten rule), and to me, vagrant falls inside the virtualization domain :). 

Can we  not move it? (At the test suite level/job group, I don't care much)